denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc

```diff
@@ -63,28 +112,30 @@ tutorial_data:
       - "<&e>You can manage everything related to chunks in the GUI."
       - "<&a>Command<&co> <&b>/claims"
     particle_guide: true
-  8:
-    hologram:
-      - "<&d><&l>Soul Forge"
-      - "<&e>Here you can forge your items with souls."
-      - "<&c>Souls can be acquired in the <&4>Savage Lands."
-    particle_guide: true
-  9:
+  16:
     hologram:
-      - "<&d><&l>The World (1/2)"
+      - "<&6><&l>The World (1/2)"
       - "<&e>The world is split into 2 sections<&co>"
-      - "<&e>The &cSavage Lands &eare the inner 20,000 blocks."
-      - "<&e>The &2Outer Realms &eare everywhere outside of that."
+      - "<&e>The <&c>Savage Lands <&e>are the inner 20,000 blocks."
+      - "<&e>The <&2>Outer Realms <&e>are everywhere outside of that."
     particle_guide: true
-  10:
+  17:
     hologram:
-      - "<&d><&l>The World Part (2/2)"
+      - "<&6><&l>The World Part (2/2)"
       - "<&e>The <&c>Savage Lands &ehave <&c>PvP Enabled."
       - "<&e>Beware of encounters there as monsters will spawn."
       - "<&e>The <&2>Outer Realms <&e>are a safe haven."
       - "<&e>No monsters will spawn, and PvP is disabled."
     particle_guide: true
-  11:
+  18:
+    hologram:
+      - "<&6><&l>Chat"
+      - "<&e>We have cross-server chat available"
+      - "<&e>Use <&6>/chat<&e> to change your channel"
+      - "<&e>You can also click on the channel name in chat!"
+      - "<&c>Please keep our rules in mind."
+    particle_guide: true
+  19:
     hologram:
       - "<&6><&l>End Game"
       - "<&e>The closer to the center of the world you get,"
```

Comment:
this line ends with a comma, should we capitalize the next line in the same sentence?